integration/simple-json-name/simple.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable */
+import { Writer, Reader } from 'protobufjs/minimal';
+
+export const protobufPackage = 'simple';
+
+export interface Simple {
+  other_name: string;
```

Comment:
Hm, my interpretation of `json_name` attribute would be that this particular name would still be `name` and then `other_name` would only show up on the JSON on the wire (i.e. in `toJson` methods) and then looked for when data is parsed in from the wire (i.e. in the `fromJson` methods).

Can we do that instead?